index.js

```diff
@@ -1,17 +1,18 @@
 'use strict';
 // TODO: Use the `URL` global when targeting Node.js 10
-const URLParser = typeof URL === 'undefined' ? require('url').URL : URL;
+const URL = require('url').URL; // eslint-disable-line prefer-destructuring
```

Comment:
This needs to stay like it is. See: https://github.com/sindresorhus/normalize-url/commit/70b5afb59a354278d98f5035067fb00dfcc4ba89